### PR TITLE
Restaurant page

### PR DESCRIPTION
**Pull request by Zubenna**
## In this pull request, I did the following,
- Set up required project files.
- Install and configured web-pack.
- Developed initial page load module.
- Developed different modules for Home, About, Menu, and Contact tabs.

### CODE REVIEW

**Review by mikenath223 — CHANGES_REQUESTED** [review]
### Hello @Zubenna  👋

**You've done pretty well here. Using es6 syntax fully was nice. 👍**


- [ ] However please refer to comments in your code for adjustments to be made.


### **STATUS: CHANGES REQUESTED** 👷‍♂️

Kindly fix these issues and request another review👍
Feel free to contact me on slack(michgolden) if you need further clarification.

HAPPY CODING👨‍💻

File: package.json [hunk 1]
```diff
@@ -17,6 +17,8 @@
   },
   "homepage": "https://github.com/Zubenna/restaurant-page#readme",
   "devDependencies": {
+    "css-loader": "^5.0.1",
+    "style-loader": "^2.0.0",
```
Comment (mikenath223) [inline package.json:23]: - [ ] Kindly include the necessary run commands in the scripts section of package.json.

````
"scripts": {
    "dev": "webpack --mode development",
    "build": "webpack --mode production",
    "start": "webpack-dev-server --mode development --open" // optional
  },

````

Comment (mikenath223) [reply package.json:23]: - Also if you'll be using the start script you'll need to include the webpack-dev-server package.-

**Review by MuriloRoque — APPROVED** [review]
## Status: Approved :green_circle: 

### Hello @Zubenna :wave: 

### Congratulations, your project is approved :clap: :clap: :clap: 

### You may merge, good luck with your next projects :rocket: 

Murilo Roque Paiva da Silva

- Github: [@muriloroque](https://github.com/MuriloRoque)
- Linkedin: [MuriloRoque](https://www.linkedin.com/in/murilo-roque-b1268741/)

[Twitter Follow](https://twitter.com/MuriloRoquePai1)

**Pull request merged**